### ISSUE TITLE
Is there anyone who succeeded?

### ISSUE THREAD

**Issue by taemin6697** [KohakuBlueleaf/HyperKohaku#17]
I can't do anything

**Comment by jiechu520** [comment]
I tried training several times without success.

**Comment by chenxinhua** [comment]
I have trained the hypernet with 1000 images, only using diffusion loss without weight supervision. I was able to achieve preliminary results,  I believe that with further experimentation and training optimization, this method should be effective!

Below are some images generated by the predicted weights of hypernet:
![hypernet](https://github.com/KohakuBlueleaf/HyperKohaku/assets/17899959/564933ad-4468-4b0a-a083-66d36f22ec34)

**Comment by taemin6697** [comment]
@chenxinhua 
Can I get a parameter or your code?

**Comment by 1585231086** [comment]
> I have trained the hypernet with 1000 images, only using diffusion loss without weight supervision. I was able to achieve preliminary results, I believe that with further experimentation and training optimization, this method should be effective!
> 
> Below are some images generated by the predicted weights of hypernet: ![hypernet](https://user-images.githubusercontent.com/17899959/269605482-564933ad-4468-4b0a-a083-66d36f22ec34.png)


Did you verify on the test data images?

**Comment by chenxinhua** [comment]
> > I have trained the hypernet with 1000 images, only using diffusion loss without weight supervision. I was able to achieve preliminary results, I believe that with further experimentation and training optimization, this method should be effective!
> > Below are some images generated by the predicted weights of hypernet: ![hypernet](https://user-images.githubusercontent.com/17899959/269605482-564933ad-4468-4b0a-a083-66d36f22ec34.png)
> 
> Did you verify on the test data images?

Yes, I performed inference on the test set.

**Comment by 1585231086** [comment]
Great work!

**Comment by taemin6697** [comment]
> 나는 1000개의 이미지로 하이퍼넷을 훈련시켰으며, 가중치 감독 없이 확산 손실만 사용했습니다. 나는 예비 결과를 얻을 수 있었고, 추가적인 실험과 훈련 최적화를 통해 이 방법이 효과적일 것이라고 믿습니다!
> 
> 다음은 하이퍼넷의 예측 가중치에 의해 생성된 일부 이미지입니다. ![하이퍼넷](https://user-images.githubusercontent.com/17899959/269605482-564933ad-4468-4b0a-a083-66d36f22ec34.png)

Is it possible to share even the weights used?

**Comment by 1585231086** [comment]
Does anyone get a convergent training loss curve? I try some changes but no effect. The gen_loss line is a random curve.

**Comment by taemin6697** [comment]
> 수렴하는 훈련 손실 곡선을 얻는 사람이 있나요? 몇 가지 변화를 시도했지만 효과가 없습니다. gen_loss 라인은 임의의 곡선입니다.

I don't now @chenxinhua

**Comment by nakulrathore97** [comment]
> I have trained the hypernet with 1000 images, only using diffusion loss without weight supervision. I was able to achieve preliminary results, I believe that with further experimentation and training optimization, this method should be effective!
> 
> Below are some images generated by the predicted weights of hypernet: ![hypernet](https://user-images.githubusercontent.com/17899959/269605482-564933ad-4468-4b0a-a083-66d36f22ec34.png)

@chenxinhua,
I wanted to know what you mean by diffusion loss here.

In the code. 
train_hyperdreambooth.py, line 1123
`                loss = weight_loss*args.pre_opt_weight_loss + img_loss`
[Link to code](https://github.com/KohakuBlueleaf/HyperKohaku/blob/765131323c31a19c2ea18ff1d2a1f505a5f7abfa/train_hyperdreambooth.py#L1123)

If "weight_loss" term is removed, only "img_loss" is used and the hypernetwork won't update.
I have verified this behaviour if only img_loss is used.

**Comment by ehedlin** [comment]
> I have trained the hypernet with 1000 images, only using diffusion loss without weight supervision. I was able to achieve preliminary results, I believe that with further experimentation and training optimization, this method should be effective!
> 
> Below are some images generated by the predicted weights of hypernet: ![hypernet](https://private-user-images.githubusercontent.com/17899959/269605482-564933ad-4468-4b0a-a083-66d36f22ec34.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA4ODE1MTQsIm5iZiI6MTcyMDg4MTIxNCwicGF0aCI6Ii8xNzg5OTk1OS8yNjk2MDU0ODItNTY0OTMzYWQtNDQ2OC00YjBhLWEwODMtNjZkMzZmMjJlYzM0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzEzVDE0MzMzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxNGQxNTg1NzY4MjdjZTgyZGI1OWVkNmUwMDUyNzcxOGVlNWUxMTIyODQyMDI2MDRlOTI4MGY4ZjFjYWNjNjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Jd2_RmBKj05w53Blzq9XXfwMDT0XUwL_908Y4jwuIKg)

Do you happen to still have the trained model weights?
